snuba/datasets/storages/functions.py

```diff
@@ -59,7 +59,7 @@
     query_processors=[],
     stream_loader=build_kafka_stream_loader_from_settings(
         processor=FunctionsMessageProcessor(),
-        default_topic=Topic.PROFILES_FUNCTIONS,
+        default_topic=Topic.PROFILES_CALL_TREE,
```

Comment:
it seems like you have some o @enochtangg 's changes here. I think you need to merge master?

Reply:
Fixed, I merged changes from his branch into mine.